ros/angel_system_nodes/angel_system_nodes/data_publishers/hl2ss_ros_bridge.py

```diff
@@ -122,8 +121,8 @@ def __init__(self):
             self.ros_frame_publisher = self.create_publisher(
                 Image, self._image_topic, 1
             )
-            self.ros_frame_ts_publisher = self.create_publisher(
-                Time, self._image_ts_topic, 1
+            self.ros_frame_md_publisher = self.create_publisher(
```

Comment:
There is a world where this is a service instead of a constantly published message, however this is definitely fine for now and is not going to exceed our bandwidth budgets.